### ISSUE TITLE
Type import error

### ISSUE THREAD

**Issue by zirho** [bviebahn/react-native-star-rating-widget#25]
Hi, 
I am getting typescript type error using the library.
I've tried to restart typescript server or reload the VS code. 
none of them worked.

![image](https://user-images.githubusercontent.com/1251296/202724533-d351f1cd-ce6f-4374-9c19-06624ce6029c.png)

```
    "typescript": "^4.8.3"
    "react-native-star-rating-widget": "^1.3.1",
    "react": "18.1.0",
    "react-native": "0.70.6",
```

Thanks you for the project

**Comment by bviebahn** [comment]
You're right, there seems to be something wrong with the types in the latest version. I'll check it out, thanks for pointing it out. For now you can use version 1.3.0 which should contain the types.

**Comment by bviebahn** [comment]
Should be fixed in version 1.4.0. Thanks for your help, let me know if there's anything else.

**Comment by zirho** [comment]
I really appreciate your time to maintain the project.
Thanks a lot!
